Update ms-cloud-germany-transition.md

bennope opened the pull request:
corrected some errors. 
1. at the end of the migration; The tenant is now a fully global Office 365 organization. (deleted not in is now not a fully global )
2. from Germany-specific products to global products (added to global products) 
3. Existing Microsoft Cloud Deutschland customers (replaced Existing Microsoft Cloud Germany (Microsoft Cloud Deutschland) customers    reason: term Microsoft Cloud Deutschland customers was already used at the beginning of the article )
4. customers can now begin to migrate .... services (replaces customers by services in: customers can now begin to migrate their Office 365, Dynamics 365 Customer Engagement, and Power Platform customers.)
5. Office for the web (replaces in MCI service plan 23. Word Online, Excel Online, PowerPoint, OneNote, and Visio Online)
6 Yammer Enterprise (replaces Yammed online in 29.)
7. Information Protection for Office 365 (replaces 11. Information Rights Management)
8. services 30. - 39. added

bennope committed:
Update ms-cloud-germany-transition.md

corrected some errors. 
1. at the end of the migration; The tenant is now a fully global Office 365 organization. (deleted not in is now not a fully global )
2. from Germany-specific products to global products (added to global products) 
3. Existing Microsoft Cloud Deutschland customers (replaced Existing Microsoft Cloud Germany (Microsoft Cloud Deutschland) customers    reason: term Microsoft Cloud Deutschland customers was already used at the beginning of the article )
4. customers can now begin to migrate .... services (replaces customers by services in: customers can now begin to migrate their Office 365, Dynamics 365 Customer Engagement, and Power Platform customers.)
5. Office for the web (replaces in MCI service plan 23. Word Online, Excel Online, PowerPoint, OneNote, and Visio Online)
6 Yammer Enterprise (replaces Yammed online in 29.)
7. Information Protection for Office 365 (replaces 11. Information Rights Management)
8. services 30. - 39. added

ghost commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/signed)](https://cla.opensource.microsoft.com/MicrosoftDocs/microsoft-365-docs?pullRequest=3751) <br/>All CLA requirements met.

bennope commented:
corrected some errors. 
1. at the end of the migration; The tenant is now a fully global Office 365 organization. (deleted not in is now not a fully global )
2. from Germany-specific products to global products (added to global products) 
3. Existing Microsoft Cloud Deutschland customers (replaced Existing Microsoft Cloud Germany (Microsoft Cloud Deutschland) customers    reason: term Microsoft Cloud Deutschland customers was already used at the beginning of the article )
4. customers can now begin to migrate .... services (replaces customers by services in: customers can now begin to migrate their Office 365, Dynamics 365 Customer Engagement, and Power Platform customers.)
5. Office for the web (replaces in MCI service plan 23. Word Online, Excel Online, PowerPoint, OneNote, and Visio Online)
6 Yammer Enterprise (replaces Yammed online in 29.)
7. Information Protection for Office 365 (replaces 11. Information Rights Management)
8. services 30. - 39. added
